plugins/unpack/7z_dir.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+#
+# $1 - unzip
+# $2 - input directory with tail slash
+# $3 - output directory with tail slash
+# $5 - archive files to delete
+# $6 - unpack temp dir
+
+ret=0
+
+process_directory()
+{
+        for fn in "$2"*.zip ; do
```

Comment:
May be we should to have .7z extension instead .zip here?

Reply:
Yes I must agree I forgot to mention that I had to change that